app/controllers/concerns/authorizable.rb

```diff
@@ -3,9 +3,19 @@ def authorize_user
     return unless current_user.phone_enabled?
 
     if user_fully_authenticated?
-      redirect_to account_url
+      add_methods_else_account_url
     elsif current_user.two_factor_enabled?
       redirect_to user_two_factor_authentication_url
     end
   end
+
+  private
+
+  def add_methods_else_account_url
```

Comment:
This method name makes me think it returns a url, similar to `user_two_factor_authentication_url`. Similar to the method by the same name in `app/controllers/concerns/two_factor_authenticatable.rb`.

Reply:
It does return a url